.github/workflows/publish.yaml

```diff
@@ -0,0 +1,24 @@
+name: Publish network lists
+on:
+  push:
+    branches: [PM-1376-add-worklow-to-publish-the-seed-list, main]
```

Comment:
make sure that branch goes away before you move this pr out of draft

Reply:
Done 👍